docs/production-deployment/cloud/terraform-provider.mdx

```diff
@@ -24,6 +24,12 @@ Temporal supports the [Cloud Namespace](/cloud/namespaces) resource management,
 
 :::
 
+:::info
+
+Currently does not support GCP.
```

Comment:
I'm not too sure about us using currently since it can date the docs. Maybe just remove the word currently.

Reply:
Such as GCP (Google Cloud Platform) is not supported by the Temporal Cloud Terraform provider.

Reply:
Fixed.